homework-submission/homeworkWeek1/app.js

```diff
@@ -0,0 +1,76 @@
+const TodoList = (props) => {
+    console.log(props)
+    return (
+        <li>
+            {props.id} {props.description}, {props.deadline} {props.done} 
+        </li>
+    )
+}
+
+const StaticList = () => {
+    return (
+        <div>
+            <ul>
+               {<TodoList description="Get out of bed" deadline="Wed Sep 13 2017" />}
+               {<TodoList description="Brush teeth" deadline="Thu Sep 14 2017" />}
+               {<TodoList description="Eat breakfast" deadline="Fri Sep 15 2017"/>}
```

Comment:
I see there is no need for any of these braces here.

Braces in React are usually used to tell the jsx compiler this will be a javascript scope and therefore javascript involves to be processed.